print("Python Job")

# Raise exception manually
# raise Exception("Manually raised exception.")